ds102/participants.json

```diff
@@ -0,0 +1,25 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
+    "Levels": {
+      "D": {
+        "Description": "D"
```

Comment:
I have removed "TermURL" here for now. If somebody knows what this means, please feel free to clarify.